app/scripts/platforms/extension.js

```diff
@@ -65,9 +65,11 @@ class ExtensionPlatform {
   }
 
   sendMessage (message, query = {}) {
-    extension.tabs.query(query, tabs => {
+    const id = query.id
+    delete query.id
```

Comment:
Why are we deleting the id from the query here?

Reply:
Querying tabs worked oddly in Chrome and Firefox.  If the `id` was not deleted before querying, no tabs would ever be returned. This is why this roundabout method of sending to a specific ID is used: if an `id` is present on a `query`, we cache it, delete it, run the rest of the `query`, then only send the resulting message to tabs matching the cached `id`.